src/routes/users.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
The 'use strict' directive is not necessary in this case, as it is automatically enabled in ES6 modules.
But don't change it here